### PR TITLE
 feat:Create Notifications Dashboard and Card Compontents

### PR DESCRIPTION
**Pull request by higueraDev**
### Summary ### 
Changes solves the issue #121 

@Basultobd

### CODE REVIEW

File: packages/notification-system/src/components/NotificationsDashboard/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import NotificationCard from '../NotificationCard';
+import styles from './NotificationsDashboard.module.scss';
+
+const NotificationsDashboard = () => {
+  return (
+    <>
+      <div className={styles.dashboard}>
```
Comment (Basultobd) [inline packages/notification-system/src/components/NotificationsDashboard/index.js:8]: Is this the view component? If yes som recomendations:

1. This is not a componente per se, this is a view, so what if we name the file `NotificationsDashobard.view` and we put it inside a `views` dir?
2. Lets simulate the request using a `useEffect` and a `setTimeout`, take look of `useState` hook too

File: yarn.lock [hunk 2]
```diff
@@ -64,6 +64,27 @@
     json5 "^2.1.2"
     semver "^6.3.0"
 
+"@babel/core@^7.17.7":
```
Comment (Basultobd) [inline yarn.lock:67]: Should this changes be added? Im seeing that is on root level?

File: packages/notification-system/src/components/NotificationsDashboard/NotificationsDashboard.module.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
+.dashboard {
+	background-color: #fff;
```
Comment (Basultobd) [inline packages/notification-system/src/components/NotificationsDashboard/NotificationsDashboard.module.scss:2]: Is there any convention related with the colors at cohort level?

**Review by Basultobd — CHANGES_REQUESTED** [review]
Some comments :)

Could you add images on how all these changes look?

**Comment by higueraDev** [comment]
> Some comments :)
> 
> Could you add images on how all these changes look?

### **Desktop**
Notification Dashboard View

### **Mobile**
Notification Dashboard View

File: yarn.lock (hunk 2 above)
Comment (higueraDev) [reply yarn.lock]: I don't figured how those changes came up. I removed them now, on changes.

File: packages/notification-system/src/components/NotificationsDashboard/NotificationsDashboard.module.scss (hunk 3 above)
Comment (higueraDev) [reply packages/notification-system/src/components/NotificationsDashboard/NotificationsDashboard.module.scss:2]: I didn't find related threads yet

File: packages/notification-system/src/components/NotificationsDashboard/index.js (hunk 1 above)
Comment (higueraDev) [reply packages/notification-system/src/components/NotificationsDashboard/index.js:8]: 1. component -> view. ✔

Comment (Basultobd) [reply packages/notification-system/src/components/NotificationsDashboard/index.js:8]: Oh I mean, that we can add suffixes indicating what a file contains, for example, for redux, we can have:

1. `something.slice.tsx`
2. `something.actions.tsx`
3. `something.reducers.tsx`

Is this clear?

**Comment by Basultobd** [comment]
> > Some comments :)
> > Could you add images on how all these changes look?
> 
> ### **Desktop**
> Notification Dashboard View
> 
> ### **Mobile**
> Notification Dashboard View

They looks terrific 🚀

File: packages/notification-system/src/components/NotificationsDashboard/index.js (hunk 1 above)
Comment (higueraDev) [reply packages/notification-system/src/components/NotificationsDashboard/index.js:8]: Oh, great!. I got it now. Ty 🙇🏻‍♂️

**Comment by higueraDev** [comment]
@Basultobd hice merge del fix que hizo el squad de admin-panel

**Review by Basultobd — APPROVED** [review]
LGTM!

**Pull request merged**